geomstats/backend/pytorch.py

```diff
@@ -11,6 +11,33 @@
 int8 = 'torch.ByteTensor'
 
 
+def indexing(x):
+    return x.type(torch.LongTensor)
+
+
+def float_to_double(x):
```

Comment:
Can we use the function "gs.cast" to cast into a specific type? The types can also be referred to as gs.float32, gs.float64, etc. (see backends)

For example: ```gs.cast(x, gs.float64)```

Reply:
Done.